src/resources/Enums.ts

```diff
@@ -653,6 +653,16 @@ export enum ResultRankingsKind {
     rankingExpression = 'ranking_expression',
 }
 
+export enum ResultRankingsRuleTypes {
```

Comment:
What's the difference between rule types and rule kinds?

Reply:
At the API level, they are practically the same. At some point we added new parameters without taking into account the existing parameters :/ 

`kind` -> `ruleTypes` have the same behaviour. They are just represented differently
`enabledStatus` -> `ruleStatuses` have different behaviour. `ruleStatuses` also take into account the group enable status.

Eventually we would like to deprecate/remove `kind` and `enabledStatus` to use the new set of parameters

Reply:
Okay 😅  How could we reduce the confusion for users of the lib? May I suggest a small JSDoc above the fields of the interfaces?

Reply:
@gdostie Added jsdoc

Reply:
thanks!